build.gradle

```diff
@@ -683,6 +709,13 @@ buildProperties.doLast {
     }
 }
 
+task nodeVersionInfo(type: NpmTask) {
+    dependsOn npmSetup
+    args = ['version']
+}
+
+npm_run_bundle.dependsOn nodeVersionInfo
```

Comment:
this command adds a dozen lines of output to the build logs showing npm/node and other build-environment info